financeager/period.py

```diff
@@ -5,6 +5,7 @@
 from dateutil import rrule
 from datetime import datetime as dt
 import re
+from abc import ABC, abstractmethod
```

Comment:
I think you mixed up your branches. Switch to the period-branch, and cherry-pick the period-change commit. Back on the config-path-branch, remove the commit by hard-resetting to the previous one.